src/js/video.js

```diff
@@ -66,26 +71,23 @@ const defaultOpts = {
 	classes: [],
 	optimumwidth: null,
 	placeholder: false,
-	placeholdertitle: false,
+	placeholderInfo: ['title'],
 	data: null
 };
 
 class Video {
 	constructor(el, opts) {
 		this.containerEl = el;
 
-		this.opts = opts || getOptionsFromDataAttributes(this.containerEl.attributes);
-
-		Object.keys(defaultOpts).forEach(optionName => {
-			if (typeof this.opts[optionName] === 'undefined') {
-				this.opts[optionName] = defaultOpts[optionName];
-			}
-		});
+		this.opts = Object.assign({}, defaultOpts, opts, getOptionsFromDataAttributes(this.containerEl.attributes));
```

Comment:
`Object.assign` should probably be in the `browserFeatures.required` array in `origami.json`.